content/components/selectpanel.mdx

```diff
@@ -190,32 +220,6 @@ By employing these loading states, users can have a better understanding of the
   </Dont>
 </DoDontContainer>
 
-### Click-to-dismiss behavior
```

Comment:
👀

Reply:
I'm okay with leaving this out. What we're going with is the true and expected nature of a dialog, which is covered in the "Closing" section of this doc. (we should put the videos back in though)